psi4/src/psi4/libmints/wavefunction.cc

```diff
@@ -607,6 +607,26 @@ void Wavefunction::set_reference_wavefunction(const std::shared_ptr<Wavefunction
     reference_wavefunction_ = wfn;
 }
 
+void Wavefunction::force_doccpi(const Dimension &doccpi) {
+    for (int h = 0; h < nirrep_; h++) {
+        doccpi_[h] = doccpi[h];
+        nalphapi_[h] = doccpi_[h] + soccpi_[h];
+        nbetapi_[h] = doccpi_[h];
+    }
+    nalpha_ = doccpi_.sum() + soccpi_.sum();
+    nbeta_ = doccpi_.sum();
+}
+
+void Wavefunction::force_soccpi(const Dimension &soccpi) {
+    for (int h = 0; h < nirrep_; h++) {
+        soccpi_[h] = soccpi[h];
+        nalphapi_[h] = doccpi_[h] + soccpi_[h];
+        nbetapi_[h] = doccpi_[h];
```

Comment:
Slight pedantic, but no need to change `beta` here.

Reply:
I've removed it. Thank you!